src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2043,7 +2046,7 @@ Json::Value NetworkOPsImp::getServerInfo (bool human, bool admin)
             if (! validation_manifest.lines().empty())
             {
                 std::string s;
-                s.reserve (188);
+                s.reserve (288);
```

Comment:
That's a thought.  I usually reserve all caps for macros.  You might consider putting this declaration inside `struct Manifest`:
```
struct Manifest {
    ...
    static std::size_t constexpr textLength = 288;
    ...
```
Then you would replace the two `288`s with `Manifest::textLength`.